app/assets/templates/profile_header_tpl.jst.hbs

```diff
@@ -1,43 +1,46 @@
-{{#if loggedIn}}
-  <div class="pull-right">
-    {{#if is_own_profile}}
-      {{!-- can't block myself, so don't check it here --}}
-      <a href="{{urlTo 'editProfile'}}" id="edit_profile" class="btn btn-primary">{{t 'people.edit_my_profile'}}</a>
-    {{else}} {{#if is_blocked}}
-      <a href="#" id="unblock_user_button" class="btn btn-danger">{{t 'people.stop_ignoring'}}</a>
-    {{else}}
-      <div class="placeholder aspect_membership_dropdown"></div>
-    {{/if}}{{/if}}
-  </div>
-{{/if}}
-
-<div id="author_info">
-  <h2>
-    <span id="name">{{name}}</span>
-    <span id="diaspora_handle">{{diaspora_id}}</span>
-    {{#if show_profile_btns}}
-      {{{sharingMessage this}}}
-    {{/if}}
-  </h2>
+<div id="author_info" class="clearfix">
+  <div class="col-sm-10">
+    <h2>
+      <span id="name">{{name}}</span>
+      <span id="diaspora_handle">{{diaspora_id}}</span>
+      {{#if show_profile_btns}}
+        {{{sharingMessage this}}}
+      {{/if}}
+    </h2>
 
-  {{#if has_tags}}
-    <div class="description">
-      <i class="entypo-tag"></i>
-      {{fmtTags profile.tags}}
-    </div>
-  {{else}}
-    {{#if is_own_profile}}
+    {{#if has_tags}}
       <div class="description">
-        <i>{{t 'profile.you_have_no_tags'}}</i>
-        <span class="add_tags">
-          <a href="{{urlTo 'editProfile'}}">{{t 'profile.add_some'}}</a>
-        </span>
+        <i class="entypo-tag"></i>
+        {{fmtTags profile.tags}}
+      </div>
+    {{else}}
+      {{#if is_own_profile}}
+        <div class="description">
+          <i>{{t 'profile.you_have_no_tags'}}</i>
+          <span class="add_tags">
+            <a href="{{urlTo 'editProfile'}}">{{t 'profile.add_some'}}</a>
+          </span>
+        </div>
+      {{/if}}
+    {{/if}}
+  </div>
+  <div class="clearfix col-sm-2">
```

Comment:
Yeah, that it. As I've move the edit button under the profile informations rather than above, the first clearfix is to prevent the edit button to overlap the post and photo link like here:

![screen shot 2015-09-17 at 00 22 07](https://cloud.githubusercontent.com/assets/3429231/9920196/4b629d26-5cd2-11e5-8d93-b14d1d7a2d4a.png)

This one is because the container of the button would be height 1:

![dc1](https://cloud.githubusercontent.com/assets/3429231/9920210/7e98cd8c-5cd2-11e5-93d8-d1c83c1db0a8.png)